test/multi_node_test.py

```diff
@@ -13,27 +13,34 @@
 from ray.test.test_utils import run_and_get_output
 
 
-def run_string_as_driver(driver_script):
+def run_string_as_driver(driver_script, blocking=True):
```

Comment:
These should be two different functions, one for the blocking case and one for the non-blocking case